### ISSUE TITLE
EQ Knobs do not do an anything to sound . 2.4bata-394

### ISSUE THREAD

**Issue by sanmannor** [mixxxdj/mixxx#12644]
### Bug Description

the EQ knobs do not do anything at all. both using a controller Pioneer ddj FLEX 4 , and in the program.  the knobs turn  in the program when engaged on controller, but the sound and wave forms are not effected. When I restore defaults in preferences  it seems to work and and cut or amplify sound. This was a fresh install and kept happening after each install. 

### Version

2.4-beta-394-gd1c0de51b0

### OS

windows 10

**Comment by ronso0** [comment]
Are you saying there are no EQs configured in Preferences -> Mixer?

> This was a fresh install and kept happening after each install.

Are you sure there was no Mixxx profile from a previous version?
You can test this (emulate a 'clean install') by renaming the existing Mixxx settings directory (i.e. backup your settings). Mixxx Will create this when being started for the first time.
Here's how to find it https://manual.mixxx.org/2.4/en/chapters/appendix/settings_directory.html


Please take a look at #10737

**Comment by sanmannor** [comment]
only happened in 2.4. If I switched back to the stable version after uninstalling, it reverted to functioning EQ. I don't remember changing eq settings in 2.4 before uninstalling it. but maybe I did, as I just uninstalled and reinstalled 2.4 and it all works... so it must have been a saved setting, and now I cant reproduce it. Maybe it was a previous build that changed settings, or i did it manually and forgot.
